asar-teensy-firmware/TestScripts/Motors/TestScript1/MOTOR.cpp

```diff
@@ -0,0 +1,108 @@
+/* Motor.cpp
```

Comment:
Remove a copy of this file and make a symlink.

Reply:
changed to symlink